### PR TITLE
Mesh com 1.5 compatibility

### PR DESCRIPTION
**Pull request by SelinaTII**
** mesh.conf --> Added BRIDGE=true to enable bridging mode (put false to disable bridge)
** mesh_com_11s.conf --> Added bridge, meshint to denote bridging mode and mesh interface
** entrypoint.sh --> Added condition: if BRIDGE = true in mesh.conf, write bridge: True, meshint: br-lan to mesh_com_11s.conf, else write bridge: False, meshint: bat0
** ConnectionMgr.py: Added attribute bridge to class ConnectionMgr which is read from mesh_com_11s.conf. In starting_mesh(), run bridge_settings() after starting mesh if bridge is true
** mesh_utils.py --> Imported yaml, added function get_mesh_int() to get the interface from meshint in mesh_com_11s.conf, assigned the value returned by get_mesh_int() as default mesh interface in arguments to other functions
** header.py --> Replaced hardcoded value of MESHINT with mesh_utils.get_mesh_int()
** continuous/utils.py --> Replaced hardcoded value of MESHINT with mesh_utils.get_mesh_int()
** continuous/ca_main.py --> Replaced hardcoded value of MESHINT with mesh_utils.get_mesh_int()
** quarantine.py --> Replaced hardcoded interface 'bat0' with mesh_utils.get_mesh_int()
** mutual.py --> Updated my_ip_mesh to be the br-lan ip after the bridge is established

Jira_Id : MSS20-125

Signed-off-by: Selina Shrestha <selina.shrestha@tii.ae>

### CODE REVIEW

**Review by martin-tii — APPROVED** [review]

File: modules/utils/docker/mesh.conf
Comment (govindsi) [inline modules/utils/docker/mesh.conf:1]: config is sourced from /opt/mesh.conf in entrypoint.
source /opt/mesh.conf

We will have to use single config for both 1.0/1.5 version. Probably we can use new config for all modes.

**Review by martin-tii — CHANGES_REQUESTED** [review]

File: modules/utils/docker/entrypoint.sh [hunk 1]
```diff
@@ -30,6 +28,7 @@ configure()
       mesh_if=$MESH_VIF
       meshVersion=$MSVERSION
       ML=$ML
+      bridge=$BRIDGE
```
Comment (SelinaTII) [inline modules/utils/docker/entrypoint.sh:31]: @govindsi  @martin-tii  I've changed it to get bridge mode from /opt/mesh.conf now

**Review by Monika-Research — APPROVED** [review]
Tested eth mode and it works fine. MCC mode needs to be enabled as well and entrypoint.sh needs to re-structured accordingly. However, it can be a new request.

**Comment by martin-tii** [comment]
DO NOT MERGE, 
This PR will be closed. 
Use https://github.com/tiiuae/mesh_com/pull/273 instead

**Comment by martin-tii** [comment]
Replaced with [PR 273](https://github.com/tiiuae/mesh_com/pull/273 )

**Pull request closed without merging**